src/Courses/CourseInterfaces.tsx

```diff
@@ -323,6 +323,7 @@ export class ProblemObject implements IProblemObject {
         additionalProblemPaths?: Array<string>;
         randomSeedSet?: number[];
     }
+    rendererData?: any;
```

Comment:
What is this doing here? It doesn't look like it's used

Reply:
It's being used in the problemIframe now that the returned data from GetQuestion is being properly typed.